.eslintignore

```diff
@@ -11,28 +9,20 @@
 /dist
 /out
 
-# Min Files
-/public/lib
-
-# No linting needed
-**/__tests__/
+# Autogenerated files
+/public/lib/*
 **/participant_responses
-**/firebase-service-account.json
 psiturkit/*_turk
-
+*.log
+*.log*
 
 # Misc
 /.husky
 /.vscode
 /.env
 .env
-.DS_Store
+/.pnp
+.pnp.js
```

Comment:
you're using modern yarn?

Reply:
No, just `npm`. But it's been the gitignore since I started on the project 🤷‍♂️